hummingbot/market/binance/binance_market.pyx

```diff
@@ -1161,6 +1169,8 @@ cdef class BinanceMarket(MarketBase):
 
         # Add 1% as a safety factor in case the prices changed while making the order.
         if notional_size < float(trading_rule.min_notional_size) * 1.01:
+            self.logger().error(f"The order value is less than the minimum order value allowed and is not placed. "
```

Comment:
This is not an error. Strategies should be allowed to calculate any size they want, and they're just using this function to check whether it makes sense to emit a certain order at this market.

Reply:
I believe the user should know why an order is not placed in the market. This is just logging to show they did not meet the notional value and that's why order is not placed.

Reply:
It's up to the strategy to decide whether that's an unrecoverable error to the user or not, not the market connector.

In general, anything we log as `.error()` should be bugs. If someone, e.g., has too little balance in his wallet s.t. the strategy cannot make orders - that's a recoverable error because it can be fixed by user action. Such a thing should be a `logger().network()` call emitted from the strategy instead.